docs/design/software-design-specification.adoc

```diff
@@ -496,14 +494,11 @@ For all repositories in scope, the artifacts in the corresponding groups in Nexu
 |===
 | SDS-SETUP-1
 | Helm chart `ods-pipeline`
-a| The Helm chart consists of three subhcharts:
+a| The Helm chart consists of two subhcharts:
```

Comment:
"subhcharts" → "subcharts"

Reply:
Thanks, fixing in next PR as GitHub has some Git issues right now ...